### ISSUE TITLE
Move idris_net from rts

### ISSUE THREAD

**Issue by melted** [idris-lang/Idris-dev#2069]
It doesn't have any dependencies on anything else in the run-time system and is just a bunch of support routines for the networking library. So why not build it together with that library?

**Comment by edwinb** [comment]
Mostly because we don't currently have any C code in prelude, base or contrib and this keeps the already complicated build process a bit simpler. I don't see any real value in moving it at the moment. Eventually it might be nice to have these libraries maintained separately.

**Comment by jfdm** [comment]
I have tagged this with *Feature Request* and *Awaiting more Information* to indicate that this is something we may want in the future, but not now.

**Comment by david-christiansen** [comment]
I don't think this is either a feature request or a situation where we need information - rather, it's a question about how the build process is put together for a library shipped with Idris. I'm actually going to close the issue, because we aren't going to move the code in question, and it can be re-opened if new information comes to light.

**Comment by melted** [comment]
Open again because the mechanism for C in contrib is needed for other stuff. And this is a good customer for that.

**Comment by melted** [comment]
The reasons for ejecting this is that it adds size and portability issues for something that is rarely needed.
